package.json

```diff
@@ -76,7 +76,8 @@
     "Miika Hänninen <miika.hanninen@gmail.com>",
     "Kevin Goslar <kevin.goslar@gmail.com>",
     "John Krull <astrom.flux@gmail.com>",
-    "Maxim Koretskiy <mr.green.tv@gmail.com>"
+    "Maxim Koretskiy <mr.green.tv@gmail.com>",
+    "David Nimmo <dnimmo@gmail.com>"
```

Comment:
Ah sorry, I was just following the instructions in the "CONTRIBUTING.md" file :)